---
layout: archive
title: "Experience"
permalink: /experience/
author_profile: true

header:
  overlay_color: "#000"
  overlay_filter: "0.5"
  overlay_image: main.jpg
---

# Full-time scientist at Finch Therapeutics: June 2018 - present
**Manager:** Rotem Gura Sadovsky  
**Team Lead:** Sonia Timberlake  
I began in my role as a Bioinformatics Research Associate on Finch Therapeutics' Data Science team in June of 2018. As part of the Data Science team, I work in an extensively collaborative environment, combining insights from my teammates, bench scientists and clinicians at Finch to perform high-throughput analysis of multi-omics microbiome data from patients in a variety of disease indications. My role, which was expanded with a promotion Bioinformatics Research Associate II in June of 2020, also means that I assist in computational analysis method development, perform end-to-end implementation and validation of bioinformatics pipelines, and orchestrate meta-analysis projects as part of the Ulcerative Colitis, Crohn's Disease, Autism Spectrum Disorder and Clostridium difficile (CP101) programs.

## Analyzing the human microbiome in IBD
Ulcerative colitis (UC) and Crohn's Disease (CD), collectively form the class of heterogeneous gastrointestinal inflammatory disorders known as inflammatory bowel disease (IBD). As part of the microbial candidate discovery process for in Finch's [Rationally Selected Microbiota](https://finchtherapeutics.com/platform) therapy in IBD, I have reviewed, accessed, and processed data dozens of public and proprietary datasets constituting thousands of samples and billions of reads for meta-analysis. This is involved building and validating version-controlled and scalable bioinformatics pipelines with high-performance parallelized architecture to quality-filter DNA sequences, remove contaminant DNA, and quantify strain abundances using both public and custom databases and methods. After processing was complete, I set out to identify biological features which strongly distinguished IBD and non-IBD cohorts using non-parametric statistics and machine learning. I corrected for correlated patient metadata features, discovered disease location and behavior-specific microbial populations and assessed phylogenetic and functional information to develop hypotheses about biological mechanisms. In addition to building outstanding technical skills and experience working with high volumes of real-world data under tight deadlines, these projects require that I work independently and effectively communicate my decisions and hypotheses as I assessed data quality, consulted literature for best practices in pipeline construction, and executed on multiple complex projects while effectively communicating my results to scientists, clinicians, and consultants with a variety of backgrounds.

## Computational and scientific skills
Without technical support for HPC resources typical in academic institutions, my team and I are the managers and developers of our own infrastructure: as a result, I have become responsible for version-controlled management of multiple pipelines using both Amazon Web Services and Google Cloud Platform architecture. I have spent thousands of hours developing code in Python and bash, as well as constructing pipelines using HPC-specific languages like WDL and SLURM. I examine and manipulate metagenomic and WGS from bacterial and non-bacterial organisms, amassing understanding of the features and caveats of each data type. Technical expertise of this magnitude would have been impossible to build during my undergraduate degree. I am extremely experienced with common data manipulation and visualization libraries like pandas, numpy, seaborn, scipy, and BioPython. At Finch I have worked with hundreds of pieces of software, including but not limited to QIIME, DADA2, mothur, the bioBakery 'omics tools, Robert C. Edgar's tools and their derivatives, bowtie2, KMA, and other mapping software, WDL/Cromwell, phylogenetic manipulation software and packages like SEPP and ete, statistical analysis frameworks and more. I am forunate to have experienced every point of a experiment's lifecycle: ideation, defining which data the project requires, receiving the data, building and validating a pipeline, examining the results, developing a scientific narrative, and iterating on findings with my coworkers and collaborators.

## Other projects
In addition to the IBD and computational platform development projects, I participate in a variety of activities that utilize my unique skillset at Finch. I optimize experimental procedures by assessing compositions of mock and actual microbial samples under a variety of protocols. I build phylogenetic models to assess 16S rRNA sequencing region specificity and strain abundance in order to guarantee isolation attempts for drug candidate microbes are successful. I construct metrics for identifying significant features in meta-analysis from a variety of contexts, and apply these metrics to actual data.


# Carleton College: 2014-2018, B.A.

My education at Carleton College involved a multidisciplinary combination of 

## Anderson lab

Examined genomically-resolved archaeal communities from the deep subseafloor using bioinformatics tools
· Conducted self-directed analysis of evolutionary dynamics with metagenomic and metatranscriptomic datasets
· Presented findings on evolutionary effects of pangenomic variation at American Geophysical Union Fall Meeting in
New Orleans, December 2017


# Teaching and work-study
## Carleton Prefect Program

## Biology Stockroom
